scripts/pd.json

```diff
@@ -5544,6 +5897,7 @@
           "dashLength": 10,
           "dashes": false,
           "datasource": "${DS_TEST-CLUSTER}",
+          "description": "The count of etcd transactions",
```

Comment:
Should it be the rate?

Reply:
The title of it is "Handle transactions count". But the target is `sum(rate(pd_txn_handle_txns_duration_seconds_count[5m])) by (instance, result)`

I will change both description and title into "rate".